packages/bull-vault/src/BullStrategy.sol

```diff
@@ -124,9 +122,8 @@ contract BullStrategy is ERC20, LeverageBull {
     }
 
     /**
-     * @notice deposit function that handle minting shares and depositing into the leverage component
-     * @dev this function assume the _from depositor already have _crabAmount
-     * @param _crabAmount amount of crab token
+     * @notice handle minting shares and rebalancing WETH/USDC leverage
```

Comment:
think this is the wrong description, right? doesn't rebalance

Reply:
yeah definitely not right!